src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,119 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private Object[] objects;
+    private int currantLength;
+    private int numberOfElements;
```

Comment:
```suggestion
    private static final int DEFAULT_CAPACITY = 10;
    private Object[] elements;
    private int size;
```
you can use `elements.length` instead of `currentLength`
add constant for default capacity